### ISSUE TITLE
Video reproduction implementation missing.

### ISSUE THREAD

**Issue by albertomcastro4** [Ancurio/mkxp#237]
![image](https://user-images.githubusercontent.com/77887431/118397211-428cda00-b653-11eb-91e2-273a4ca956ad.png)

**Comment by Ancurio** [comment]
yes, I had planned on implementing this since at least 2014 (sigh).
https://icculus.org/theoraplay/ is my preferred solution to this.

**Comment by cremno** [comment]
Alternatively https://github.com/AprilAndFriends/theoraplayer which is also open source (BSD license) and used by the original implementation (see RPGVXA's About window).

**Comment by Speak2Erase** [comment]
Hey, is this ever going to get implemented?
I'd try it myself but I sorta lack the experience

**Comment by Ancurio** [comment]
I intend on writing up a big "road to 1.0" post soon, and this item is definitely going to be up there on the list. So yeah, it is a priority.

**Comment by Speak2Erase** [comment]
Alrighty!
